src-gui/Makefile

```diff
@@ -2,34 +2,36 @@ APP := openkilda-gui
 
 rebuild: clean clean-java build
 
-build: clean target/${APP}.jar
+build:  clean-cache build/libs/${APP}.war
```

Comment:
Why do we need to clean-cache on each build? This makes impossible incremental builds and forces a developer to wait each time. Can we reconsider this?

Reply:
I have already created a task for this. #3734.